core/src/main/java/io/questdb/MessageBusImpl.java

```diff
@@ -446,37 +446,20 @@ public MCSequence getWalTxnNotificationSubSequence() {
 
     @TestOnly
     public void clear() {
-        columnPurgePubSeq.clear();
         columnPurgeSubSeq.clear();
-        groupByMergeShardPubSeq.clear();
         groupByMergeShardSubSeq.clear();
-        indexerPubSeq.clear();
         indexerSubSeq.clear();
-        latestByPubSeq.clear();
         latestBySubSeq.clear();
-        columnTaskPubSeq.clear();
         columnTaskSubSeq.clear();
-        o3CopyPubSeq.clear();
         o3CopySubSeq.clear();
-        o3OpenColumnPubSeq.clear();
         o3OpenColumnSubSeq.clear();
-        o3PartitionPubSeq.clear();
         o3PartitionSubSeq.clear();
-        o3PurgeDiscoveryPubSeq.clear();
         o3PurgeDiscoverySubSeq.clear();
-        tableWriterEventPubSeq.clear();
         textImportColSeq.clear();
-        textImportPubSeq.clear();
-        textImportRequestPubSeq.clear();
         textImportRequestSubSeq.clear();
         textImportSubSeq.clear();
-        vectorAggregatePubSeq.clear();
         vectorAggregateSubSeq.clear();
-        walTxnNotificationPubSequence.clear();
         walTxnNotificationSubSequence.clear();
-        for (int i = 0, n = pageFrameReducePubSeq.length; i < n; i++) {
-            pageFrameReducePubSeq[i].clear();
-        }
         for(int i = 0, n = pageFrameReduceSubSeq.length; i < n; i++) {
```

Comment:
nit: auto-formatter seems not to kick in here:
```suggestion
        for (int i = 0, n = pageFrameReduceSubSeq.length; i < n; i++) {
```

Reply:
fixed here: https://github.com/questdb/questdb/pull/4400/commits/6b2e03f4dc137ae17739a0b096db1bd21ea6ee6a